frontend/app/initializers/current-user.js

```diff
@@ -1,19 +1,13 @@
-import currentUserProxy from '../services/current-user.js';
-
 export function initialize(container, app) {
-  var user, store = container.lookup('store:main');
-
-  app.register('service:current-user', currentUserProxy);
```

Comment:
They made it so you can inject services into objects around that version with syntax like:
`currentUser: Ember.service.inject()`

but this is just ember-resolver magic.